sources/metricdog/README.md

```diff
@@ -0,0 +1,55 @@
+# metricdog
+
+Current version: 0.1.0
+
+## Introduction
+
+Metricdog sends anonymous information about the health of a Bottlerocket host.
+It does so by sending key-value pairs as query params in an HTTP GET request.
+
+Metricdog also has the ability to check that a list of critical services is running.
+It does so using `systemctl` and reports services that are not healthy.
+
+## What it Sends
+
+#### The standard set of metrics:
+
+* `sender`: the application sending the report.
+* `event`: the event that invoked the report.
+* `version`: the Bottlerocket version.
+* `variant`: the Bottlerocket variant.
+* `arch`: the machine architecture, e.g.'x86_64' or 'arm'.
+* `aws_region`: the AWS region the machine is running in.
```

Comment:
Does this need to be AWS specific?  As far as I can tell, we could just leave this as "region" and be a bit more agnostic so other platforms can make use of this field as well

Reply:
If it were running on another platform that had the same region name, we'd need some way to tell the difference, and I don't think the other fields would help.  Another option would be a "platform" field set to "aws", and a "region" field?

Reply:
TL;DR: I agree `region` is better than `aws_region`, and I would prefer not to add a field such as `platform=aws`.

I initially called this field `region` and I strongly agree with changing it back to `region`. My thinking is that `region` is general-enough of a concept to be potentially useful with any cloud provider, and even for on-prem deployments, since it really just means 'general location'. In fact, I dislike calling `aws_region` so much that the current version of the backend accepts either `region` or `aws_region`! I should have spoken up sooner.

My disinclination to include `platform=aws` is less strong, but it feels unnecessary since the `variant` value conveys that information, and especially because we have not yet decided how we plan to codify such 'attributes of variant' in the system itself.

Reply:
I don't think variant will convey all of the information we need.  aws-dev can run anywhere, but may only be broken on one platform, for example.

Reply:
Re: the region field - let's change it to be `region`; we all seem to agree on that.

Thinking about the `platform` field - I think it's a good idea, but it raises some questions like "How do we know what platform we're on?, etc"  

I don't think we need to block on this, and if we choose to ship without the changes, let's open an issue to track fixing this for other platforms.